components/TopBar.js

```diff
@@ -115,7 +115,7 @@ const TopBar = ({ showSearch, menuItems, showProfileAndChangelogMenu, account, n
     >
       <Link href="/">
         <Flex alignItems="center">
-          <Image width="36" height="36" src="/static/images/opencollective-icon.png" alt="Open Collective" />
+          <Image width={32} height={32} src="/static/images/oc-logo-watercolor-256.png" alt="Open Collective" />
```

Comment:
This image while actually larger in pixels (256x256, 92KB) than the current icon used (141x144, 30KB), becomes equal in size with Next.js automatic static image optimization, but it has the correct aspect ratio.

new:
<img width="403" alt="Screenshot 2023-09-11 at 13 45 12" src="https://github.com/opencollective/opencollective-frontend/assets/1552194/d1e956e1-dfaa-4a93-8748-dcd0f5401bac">

current:
<img width="399" alt="image" src="https://github.com/opencollective/opencollective-frontend/assets/1552194/8e3f174f-7b48-4504-9a23-85b3670a9093">